package/src/native.ts

```diff
@@ -316,97 +296,139 @@ type Handlers = {
-    FlatList = handlers.FlatList;
+    NativeHandlers.FlatList = handlers.FlatList;
   }
 
   if (handlers.getLocalAssetUri !== undefined) {
-    getLocalAssetUri = handlers.getLocalAssetUri;
+    NativeHandlers.getLocalAssetUri = handlers.getLocalAssetUri;
   }
 
   if (handlers.getPhotos !== undefined) {
-    getPhotos = handlers.getPhotos;
+    NativeHandlers.getPhotos = handlers.getPhotos;
   }
 
   if (handlers.iOS14RefreshGallerySelection !== undefined) {
-    iOS14RefreshGallerySelection = handlers.iOS14RefreshGallerySelection;
+    NativeHandlers.iOS14RefreshGallerySelection = handlers.iOS14RefreshGallerySelection;
   }
 
   if (handlers.oniOS14GalleryLibrarySelectionChange !== undefined) {
-    oniOS14GalleryLibrarySelectionChange = handlers.oniOS14GalleryLibrarySelectionChange;
+    NativeHandlers.oniOS14GalleryLibrarySelectionChange =
+      handlers.oniOS14GalleryLibrarySelectionChange;
   }
 
   if (handlers.pickDocument !== undefined) {
-    pickDocument = handlers.pickDocument;
+    NativeHandlers.pickDocument = handlers.pickDocument;
   }
 
   if (handlers.pickImage !== undefined) {
-    pickImage = handlers.pickImage;
+    NativeHandlers.pickImage = handlers.pickImage;
   }
 
   if (handlers.saveFile !== undefined) {
-    saveFile = handlers.saveFile;
+    NativeHandlers.saveFile = handlers.saveFile;
   }
 
   if (handlers.SDK) {
-    SDK = handlers.SDK;
+    NativeHandlers.SDK = handlers.SDK;
   }
 
   if (handlers.shareImage !== undefined) {
-    shareImage = handlers.shareImage;
+    NativeHandlers.shareImage = handlers.shareImage;
   }
 
   if (handlers.Sound) {
-    Sound = handlers.Sound;
+    NativeHandlers.Sound = handlers.Sound;
   }
 
   if (handlers.takePhoto !== undefined) {
-    takePhoto = handlers.takePhoto;
+    NativeHandlers.takePhoto = handlers.takePhoto;
   }
 
   if (handlers.triggerHaptic !== undefined) {
-    triggerHaptic = handlers.triggerHaptic;
+    NativeHandlers.triggerHaptic = handlers.triggerHaptic;
   }
 
   if (handlers.Video !== undefined) {
-    Video = handlers.Video;
+    NativeHandlers.Video = handlers.Video;
   }
 
   if (handlers.setClipboardString !== undefined) {
-    setClipboardString = handlers.setClipboardString;
+    NativeHandlers.setClipboardString = handlers.setClipboardString;
   }
 };
 
-export const isImagePickerAvailable = () => !!takePhoto;
-export const isDocumentPickerAvailable = () => !!pickDocument;
-export const isClipboardAvailable = () => !!setClipboardString;
-export const isVideoPlayerAvailable = () => !!Video;
-export const isHapticFeedbackAvailable = () => !!triggerHaptic;
-export const isShareImageAvailable = () => !!shareImage;
-export const isFileSystemAvailable = () => !!saveFile || !!deleteFile;
-export const isAudioRecorderAvailable = () => !!Audio?.startRecording;
-export const isSoundPackageAvailable = () => !!Sound.Player || !!Sound.initializeSound;
+export const isImagePickerAvailable = () => !!NativeHandlers.takePhoto;
```

Comment:
These helpers can be directly imported, as they're using the referenced `NativeHandlers` object directly (and their return value will mutate with it)